EventFilter/Utilities/plugins/EvFFEDExcluder.cc

```diff
@@ -0,0 +1,48 @@
+#include <memory>
+#include <algorithm>
+
+#include "EvFFEDExcluder.h"
+#include <FWCore/Framework/interface/Event.h>
+#include <FWCore/ParameterSet/interface/ConfigurationDescriptions.h>
+#include <FWCore/ParameterSet/interface/ParameterSet.h>
+#include <FWCore/ParameterSet/interface/ParameterSetDescription.h>
+#include <FWCore/Utilities/interface/InputTag.h>
+#include <DataFormats/FEDRawData/interface/FEDNumbering.h>
+#include <DataFormats/FEDRawData/interface/FEDRawDataCollection.h>
```

Comment:
```suggestion
#include "FWCore/Framework/interface/Event.h"
#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
#include "FWCore/ParameterSet/interface/ParameterSet.h"
#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
#include "FWCore/Utilities/interface/InputTag.h"
#include "DataFormats/FEDRawData/interface/FEDNumbering.h"
#include "DataFormats/FEDRawData/interface/FEDRawDataCollection.h"
```

Reply:
Fixed in the latest version. By the way, is there an advantage using `"` in this case, or is this more of a convention?